Mathlib/Tactic/Attr/Register.lean

```diff
@@ -87,3 +87,12 @@ register_label_attr is_poly
 
 /-- A simp set for the `fin_omega` wrapper around `omega`. -/
 register_simp_attr fin_omega
+
+/-- A simp set for simplifying expressions involving `⊤` in `enat_to_nat`. -/
+register_simp_attr enat_to_nat_top_simps
```

Comment:
Slight preference for these just to be called `enat_to_nat`, etc. This will align with the naming scheme I'm using in Lean.

Reply:
Scratch that, I missed that there were two simp sets.

I prefer removing the `_simps`, I guess -- this is implicit in it being a simp_attr.